src/util/Serializer/TripleSerializer.h

```diff
@@ -172,14 +198,12 @@ inline std::tuple<LocalVocab, std::vector<std::vector<Id>>> deserializeIds(
   AD_LOG_INFO << "Reading and processing persisted updates from " << path
               << " ..." << std::endl;
   detail::readHeader(serializer);
```

Comment:
Since this is a breaking change, you should increment the version number of the protocol to prevent people from using outdated formats for this.